### PR TITLE
First task-deviding list into pages

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
Fixed the problem with not being able to handle the last elements not being enough for entire page

### CODE REVIEW

**Comment by mgenov** [comment]
LGTM !

**Comment by KrasimirRaikov** [comment]
is the second task OK?

**Comment by mgenov** [comment]
Why this didn't happen in following PR ?

File: src/com/cloway/sentences/Sentence.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+/**
+ * @author raikov.krasimir@gmail.com (Krasimir Raikov)
+ */
+public class Sentence {
+
+  /**
+   * Compiles a HashMap of how much times each word occurs in a string
```
Comment (mgenov) [inline src/com/cloway/sentences/Sentence.java]: The method doesn't compile HashMap.

The method just counts the number of occurrences of each word in the provided sentence. It returns a map which uses word as a key and number of occurrences as value.

File: src/com/cloway/sentences/Sentence.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    return map;
+  }
+
+  /**
+   * Retern's the most common character for a string
+   *
+   * @param sentence from which to search for the most frequent character
+   * @return a Map with one element which is the character most frequently occurring in a string
+   */
+  public Map<Character, Integer> charMostFrequent(String sentence) {
```
Comment (mgenov) [inline src/com/cloway/sentences/Sentence.java]: mostFrequentCharacter sounds like a better name

**Comment by KrasimirRaikov** [comment]
I forgot that when I commit to my fork it also commits in the pull request

File: src/com/cloway/sentences/Sentence.java (hunk 2 above)
Comment (mgenov) [inline src/com/cloway/sentences/Sentence.java]: Why you are returning map when you have single character as result?

Comment (KrasimirRaikov) [inline src/com/cloway/sentences/Sentence.java]: because of the number of occurrences

**Comment by KrasimirRaikov** [comment]
Is it ok now?

**Comment by mgenov** [comment]
LGTM !

**Comment by KrasimirRaikov** [comment]
should I merge the pull request?

**Comment by mgenov** [comment]
LGTM = looks good to me, which means that you are authorised to merge it, but don't use the Merge button. 

git push upstream master:krasimir 

//push your local master to krasimir in upstream.

**Pull request merged**